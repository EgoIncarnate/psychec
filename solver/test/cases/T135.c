void path()
{
    G->adj[i]->v = 1;
}
