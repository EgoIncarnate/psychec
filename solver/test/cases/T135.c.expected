struct __T1__ { int v; };
struct __T0__ {struct __T1__** adj;}* G ;
int i ;
